modules/oauth-default-clients.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-oauth-clients.adoc
+
+[id="oauth-default-clients_{context}"]
+= Default OAuth clients
+
+The following OAuth clients are automatically created when starting the {product-title} API:
```

Comment:
same as above, I don't think you need to say "automatically" (esp. since the API suggests it is done programmatically anyway)